### ISSUE TITLE
[Bug]: handling of `load` reserved word

### ISSUE THREAD

**Issue by sherbert** [babelfish-for-postgresql/babelfish_extensions#137]
### Contact Details

herbert@dc-energy.com

### What happened?

`load` is on the T-SQL [list of reserved words](https://docs.microsoft.com/en-us/sql/t-sql/language-elements/reserved-keywords-transact-sql?view=sql-server-ver15), but is actually accepted in some contexts on SQL Server 2019 (the only version I have to test at the moment). Example:
```
1> SELECT @@VERSION
2> GO


------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Microsoft SQL Server 2019 (RTM-GDR) (KB4583458) - 15.0.2080.9 (X64)
        Nov  6 2020 16:50:01
        Copyright (C) 2019 Microsoft Corporation
        Express Edition (64-bit) on Windows 10 Enterprise 10.0 <X64> (Build 19044: )


(1 rows affected)
1> CREATE DATABASE foo
2> GO
1> USE foo
2> GO
Changed database context to 'foo'.
1> CREATE TABLE dbo.load (bar INT)
2> GO
1> SELECT * FROM dbo.load
2> GO
bar
-----------

(0 rows affected)
1>
```

However, this fails in Babelfish 1.2.0:
```
1> SELECT @@VERSION
2> GO
version
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Babelfish for PostgreSQL with SQL Server Compatibility - 12.0.2000.8
Apr 15 2022 11:39:04
Copyright (c) Amazon Web Services
PostgreSQL 13.5 Babelfish for PostgreSQL on x86_64-pc-linux-gnu

(1 rows affected)
1> SELECT CAST(SERVERPROPERTY('BabelfishVersion') AS VARCHAR)
2> GO
serverproperty
------------------------------
1.2.0

(1 rows affected)
1> CREATE DATABASE foo
2> GO
1> USE foo
2> GO
Changed database context to 'foo'.
1> CREATE TABLE dbo.load (bar INT)
2> GO
Msg 33557097, Level 16, State 1, Server BABELFISH, Line 1
syntax error near 'load' at line 1 and character position 17
1>
```

This may not be a bug since I'm not sure whether the goal is compatibility with T-SQL the spec or SQL Server the product, but just reporting since I noticed it.

### Version

BABEL_1_X_DEV (Default)

### Extension

babelfishpg_tsql (Default)

### What flavour of Linux are you running into the bug?

Ubuntu (Default)

### Relevant log output

```shell
2022-04-22 17:40:04.502 EDT [1191124] LOG:  Unmapped error found. Code: 16801924, Message: syntax error near 'load' at line 1 and character position 17, File: tsqlIface.cpp, Line: 2128, Context: babelfishpg_tsql
2022-04-22 17:40:04.502 EDT [1191124] CONTEXT:  TDS Protocol: Message Type: SQL BATCH, Phase: TDS_REQUEST_PHASE_PROCESS. Processing SQL Batch Request
2022-04-22 17:40:04.502 EDT [1191124] ERROR:  syntax error near 'load' at line 1 and character position 17 at character 17
2022-04-22 17:40:04.502 EDT [1191124] QUERY:  CREATE TABLE dbo.load (bar INT)

2022-04-22 17:40:04.502 EDT [1191124] CONTEXT:  TDS Protocol: Message Type: SQL BATCH, Phase: TDS_REQUEST_PHASE_PROCESS. Processing SQL Batch Request
2022-04-22 17:40:04.502 EDT [1191124] LOG:  Unmapped error found. Code: 16801924, Message: syntax error near 'load' at line 1 and character position 17, File: tsqlIface.cpp, Line: 2128, Context: TDS
2022-04-22 17:40:04.502 EDT [1191124] CONTEXT:  TDS Protocol: Message Type: SQL BATCH, Phase: TDS_REQUEST_PHASE_PROCESS. Processing SQL Batch Request
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by robverschoor** [comment]
We've seen similar cases before, it's a grammar-level issue. I will put it on the to-do list.

**Comment by sangilsong** [comment]
We already added all the missing unreserved keywords to a grammar file correctly before. But 'LOAD' seems like a different case.

The sqlserver document says LOAD is a reserved keyword and it can be used as an identifier only when it is "delimited" (https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/language-elements/reserved-keywords-transact-sql.md) but actually it works differently from the document.

**Comment by suprio-amzn** [comment]
Issue has been fixed and fix is available in the Babelfish 2.10 release - see [https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/tag/BABEL_2_1_0__PG_14_3](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/tag/BABEL_2_1_0__PG_14_3).
